docker/run.sh

```diff
@@ -0,0 +1,17 @@
+# For systems without journald.
+mkdir -p /var/log/journal
+
+# Copy host libsystemd into image to avoid compatibility issues. This operation
+# is only performed if the host's lib is mounted into the container through a 
+# volume such as -v /usr/lib64/:/host/lib/.
+if [ -n "$(ls /host/lib/libsystemd* 2>/dev/null)" ]; then
+  rm /lib/x86_64-linux-gnu/libsystemd*
+  ln -s /host/lib/libsystemd* /lib/x86_64-linux-gnu/
+fi
+
+if [ -n "$METADATA_AGENT_HOSTNAME" ]; then
+  sed -i "s/local-metadata-agent.stackdriver.com/$METADATA_AGENT_HOSTNAME/" \
+    /etc/google-fluentd/google-fluentd.conf
+fi
+
+/usr/sbin/google-fluentd $@
```

Comment:
Forgot: let's quote this (i.e., `"$@"`). That way arguments with spaces will be processed correctly.
This isn't going to break anything now, since you're invoking `run.sh` with no arguments, but it's the kind of thing that's easy to miss, so better to fix it.